.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+    "extends": [
+    "airbnb",
+    "prettier",
+    "prettier/react",
+    "plugin:prettier/recommended",
+    "eslint-config-prettier"
+    ],
+    "parser": "babel-eslint",
+    "rules": {
+    "import/no-unresolved": "off",
+    "react/jsx-filename-extension": [
+    1,
+    {
+    "extensions": [".js", ".jsx"]
+    }
+    ],
+    "prettier/prettier": [
+    "error",
+    {
+    "trailingComma": "es5",
+    "singleQuote": true,
+    "printWidth": 100
+    }
+    ]
+    },
+    "plugins": ["prettier"]
+    }
```

Comment:
I have seen some examples that setup the airbnb linter with some minimal configuration. I notice here is also adding Prettier, it's ok, but are you overriding some rules of the linter? Also, keep in mind the alignment.

Reply:
The plugin is added so that Prettier, as a formatter extension for VSCode, can format the code using the previously configured Linter of choice. 

If we were to override it, the configuration asks to add it in the extends array of the eslintrc.json file so that prettier can overlook some of the conflicting linter rules, which we don't want.

Reference:
https://prettier.io/docs/en/integrating-with-linters.html

Reply:
Ok, the original eslitrc.json file that gets created when you install it has very few differences than the one here, it all depends on the selected configuration when you initialize ESlint by running it for the first time in the project; you can select a popular coding style, where you'll be using the linter, where is the code going to run, and based on these, the configuration file will be created accordingly. 

For the rest of the plugins like react or prettier, its added specifically for this case.
For example, if we were to install it on its own without the react plugin or the prettier configuration we wouldn't have those objects in the array.